pkgs/development/python-modules/django-markdownx/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, django
+, fetchFromGitHub
+, markdown
+, pillow
+, python
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "django-markdownx";
+  version = "4.0.7";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "neutronX";
+    repo = "django-markdownx";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-FZPUlogVd3FMGeH1vfKHA3tXVps0ET+UCQJflpiV2lE=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "install_requires=get_requirements()," ""
+  '';
```

Comment:
Why do we need to remove this line?

Reply:
it is trying to install dependencies via pip

Reply:
I tried to build it on my machine and it seems to work fine without this patch.